radlibDemoApp/platforms/android/src/com/radlib/cordova/BluetoothComm.java

```diff
@@ -164,7 +164,7 @@ public void onReceive(Context context, Intent intent) {
 	            	deviceNamesArray[deviceNames.size()] = "Cancel";
 	            	AlertDialog.Builder builder = new AlertDialog.Builder(cordova.getActivity());
 					final AlertDialog dialogue = builder.create();
```

Comment:
If you're turning tabs into spaces, while you're at it - this seems a little too indented here...